spec/rubocop/cop/rspec/subject_stub_spec.rb

```diff
@@ -321,4 +321,46 @@
       end
     RUBY
   end
+
+  it 'flags when there are several top level example groups' do
+    expect_offense(<<-RUBY)
+      describe Foo do
+        subject(:foo) { described_class.new }
+
+        specify do
+          expect(foo).to eq(foo)
+        end
+      end
+
+      describe Bar do
+        subject(:bar) { described_class.new }
+
+        specify do
+          allow(bar).to receive(:bar).and_return(baz)
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Do not stub methods of the object under test.
+        end
+      end
+    RUBY
+  end
+
+  describe 'top level example groups' do
+    %i[
+      describe xdescribe fdescribe
+      context xcontext fcontext
+      feature xfeature ffeature
+      example_group
+      shared_examples shared_examples_for shared_context
+    ].each do |method|
+      it "flags in top level #{method}" do
+        expect_offense(<<-RUBY)
+          #{method} '#{method}' do
+            it 'uses an implicit subject' do
+              expect(subject).to receive(:bar).and_return(baz)
```

Comment:
This kind of goes against https://github.com/rubocop-hq/rubocop-rspec/pull/226/files#diff-bb66c4002cf82dad36a808e5e54c7475R11. Maybe remove the `and_return` part if it's not strictly necessary?